recipes/open-fonts/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+openfonts="$PREFIX/fonts/open-fonts"
+mkdir -p "$openfonts" || exit 1
```

Comment:
It certainly would make sence to split it to separate fonts packages but who would do it and when? There are 20 fonts in this package! I'm not eager to do it right now. May be once per month or something like that. But not now. I'd like to add it all to a separate folder (`fonts/open-fonts/`) instead of putting it to `fonts/` where it can conflict with per font packages.

Reply:
Thanks for `set -euo pipefail` and `IFS=$'\n\t'` suggestion! I'll use it here.

Reply:
I think it is fine.  This is actually packaging a separate project (which just happens to bundle fonts)